client/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Button from './Button';
+
+export default {
+  title: 'Button',
+};
+export const Default = () => (
+  <Button variant="transparent"> Button </Button>
+);
+export const Primary = () => (
+  <Button variant="blue">  Button </Button>
```

Comment:
The idea behind storybook is to see all your components and how you can customize them.

Here for the Primery/Blue buttons we have only the normal styling. What about `loading` and `disabled`?
For each variant we want to add these as well, so you see the entire picture.

Something like:
```
export const Primary = () => (
  <>
    <Button variant="blue">  Blue Button</Button>
    <Button variant="blue" disabled>  Disabled Blue Button</Button>
    <Button variant="blue" loading>  Loading Blue Button</Button>
  </>
)
```